src/client/Microsoft.Identity.Client/Instance/Region/RegionManager.cs

```diff
@@ -160,7 +160,7 @@ private async Task<RegionInfo> DiscoverAndCacheAsync(string azureRegionConfig, I
 
         private async Task<RegionInfo> DiscoverAsync(ICoreLogger logger, CancellationToken requestCancellationToken)
         {
-            string region = Environment.GetEnvironmentVariable("REGION_NAME");
+            string region = Environment.GetEnvironmentVariable("REGION_NAME")?.Replace(" ", string.Empty)?.ToLower();
```

Comment:
Please add test that shows that this transformation is correct.

Reply:
@SameerK-MSFT, what happens if the environment variable does not exist?

Reply:
I just tested and it looks like @SameerK-MSFT  is right. According to the docs https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and- the null-conditional operator is short-circuiting, so the `ToLower()` never executes. 

Great question, I didn't know about this.

Reply:
Thanks @SameerK-MSFT and @bgavrilMS : I did not know that was the way it worked!